torch_ort_inference/torch_ort/ortinferencemodule/__init__.py

```diff
@@ -33,7 +33,6 @@ def _defined_from_envvar(name, default_value, warn=True):
 # NOTE: To *change* values in runtime, import onnxruntime.training.ortmodule and
 # assign them new values. Importing them directly do not propagate changes.
 ################################################################################
-ONNX_OPSET_VERSION = 14
 MINIMUM_RUNTIME_PYTORCH_VERSION_STR = "1.8.1"
```

Comment:
Should we limit the Minimum PyTorch version to 1.12?

Reply:
Noted. Updated PR.